packages/manager/apps/dedicated/client/app/account/user/ip/restriction/user-ip-restriction.controller.js

```diff
@@ -156,5 +156,9 @@ angular
         $scope.ipRestrictionCurrentEdit = null;
         $scope.ipRestrictionCurrentEditBack = null;
       };
+
+      $scope.canRemoveIp = function canRemoveIp() {
+        return $scope.defaultRule.rule === 'deny' && $scope.list.length > 1;
```

Comment:
shouldn't it be 
```suggestion
        return $scope.defaultRule.rule !== 'deny' || $scope.list.length > 1;
```
? (according to the content added )

Reply:
Yes! I've replaced `!== 'deny'` to `=== 'accept'`.